src/libstd/sys/windows/time.rs

```diff
@@ -14,7 +11,7 @@ const INTERVALS_PER_SEC: u64 = NANOS_PER_SEC / 100;
 
 #[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Debug, Hash)]
 pub struct Instant {
-    t: c::LARGE_INTEGER,
+    t: u64,
```

Comment:
<3 this idea!

It's basically trading away an extra u32 of memory, per Instant, in exchange for getting better range and same-or-better precision. IMO that's the right call here, since it:

* Eliminates the mental-model difference between Instant and Duration number types — we can treat them like natural numbers, where the distance metric is in the same units as the values and the set bounds are identical between values and distances.
* Lets us normalize the math across all platforms, if we make that change everywhere
* Maybe even lets us simplify this code a fair bit — we might be able to shrink the Instant part of the interface imported from `sys/${platform}/time.rs` down to just a `now` function.

Reply:
> we might be able to shrink the Instant part of the interface imported from sys/${platform}/time.rs down to just a now function.

Definitely!